src/common/Pagination/index.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import Button from "./Button";
+import Icon from "./Icon";
+import { Footer, Paragraph, PageText, PageCounter } from './styled';
+
+const Pagination = () => {
+  return (
+    <Footer>
+      <Button
+        previous
+        buttonText="First"
+        disabled={true}
+        body={
+          <>
+            <Icon
+              mobile
+              previous
+              disabled={true}
+              alt="back to first page symbol"
+            />
+            <Icon
+              previous
+              disabled={true}
+              alt="back to previous page symbol"
+            />
+          </>}
+      />
+      <Button
+        previous
+        buttonText="Previous"
+        disabled={true}
+        body={
+          <>
+            <Icon
+              previous
+              disabled={true}
+              alt="back to first page symbol"
+            />
+          </>}
+      />
+      <Paragraph>
+        <PageText>
+          Page
+        </PageText>
+        <PageCounter>
+          1
+        </PageCounter>
+        <PageText>
+          of
+        </PageText>
+        <PageCounter>
+          500
+        </PageCounter>
+      </Paragraph>
```

Comment:
Zastanowiłbym się nad wydzieleniem tego do osobnego komponentu ;)

Reply:
ok przy okazji robienia nowej gałęzi to dodam